<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Додати замовлення</string>
  </property>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Відмінити</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Довжина (см)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина (см)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Висота (см)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Матеріал</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accept">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Прийняти</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_length">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_width">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_height">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_count">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_material">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string>&lt;Оберіть матеріал&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>&lt;Оберіть матеріал&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Гофрокартон</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Палітурний картон</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Деревина</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Біорозкладний матеріал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пластик</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
